src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,21 @@
-export const UserInfo = () => {};
+import React from 'react';
+import './UserInfo.scss';
+
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
```

Comment:
Consider destructuring `user` object in function arguments for better readability and clarity. 🤓